src/Sylius/Bundle/AttributeBundle/Resources/public/js/sylius-attribute.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+(function ($) {
+    'use strict';
+
+    function calculateNextIndex() {
+        var nextIndex = 0;
+        var attributeItems = $('#attributes > div .collection-item');
```

Comment:
I guess we should not rely on ids, as I might want to have two or more independent attributes lists on one page.